lms/templates/courseware/info.html

```diff
@@ -73,7 +73,11 @@ <h2 class="hd hd-2 page-title">${_("Welcome to {org}'s {course_name}!").format(o
       </div>
       <div class="info-wrapper">
       % if user.is_authenticated():
-        <section class="updates">
+          % if getattr(course, 'language'):
```

Comment:
Hey @sarahkf I think we need to move the lang attribute up to this element's parent, the ```<div class=container>``` on [line 61](https://github.com/edx/edx-platform/pull/15219/files#diff-0d48d661f5b8a2bb04e9be929237da28R61). We can reliably expect that the content in the sidebar, which is authored in Studio, is in the same language as the course content.